### PR TITLE
docs(roadmap): add January 2016 meeting + archive

### PR DESCRIPTION
**Pull request by sgoings**

### CODE REVIEW

**Comment by sgoings** [comment]
I couldn't find the next open roadmap session on the public calendar... when is that scheduled?

File: docs/roadmap/planning.rst [hunk 1]
```diff
@@ -30,12 +30,24 @@ Major decisions affecting the Roadmap are discussed during Release Planning Meet
...
-The next public release planning meeting for Deis will take place on
-**Thursday, January 7th** at **12pm MDT/1900 GMT**. The `event`_ can be added
-directly to your calendar.
+We're still taking feedback and suggestions on what our users would like to
+see in the 1.x LTS release. Feel free to provide your input on the
+`LTS release issue`_ on GitHub.
+
+Over the last month we've been hard at work on Deis v2 which puts the Deis
+workflow people have come to know and love on top of the `Kubernetes`_ platform.
+Aaron walks through a demo of what we've done so far with the release of Deis v2
```
Comment (krancour) [inline docs/roadmap/planning.rst]: I would change this to "Aaron demoed what we've done so far with the release of Deis v2 Alpha."

Comment (arschles) [inline docs/roadmap/planning.rst]: Also can you add a link to the demo outline here? https://gist.github.com/arschles/5b7a75a50938913d3eb1

Comment (sgoings) [inline docs/roadmap/planning.rst]: Done.

File: docs/roadmap/planning.rst [hunk 2]
```diff
@@ -30,12 +30,24 @@ Major decisions affecting the Roadmap are discussed during Release Planning Meet
...
-**Thursday, January 7th** at **12pm MDT/1900 GMT**. The `event`_ can be added
-directly to your calendar.
+We're still taking feedback and suggestions on what our users would like to
+see in the 1.x LTS release. Feel free to provide your input on the
+`LTS release issue`_ on GitHub.
+
+Over the last month we've been hard at work on Deis v2 which puts the Deis
+workflow people have come to know and love on top of the `Kubernetes`_ platform.
+Aaron demoed what we've done so far with the release of Deis v2 Alpha.
+(`v2 alpha walkthrough cheatsheet`_)
```
Comment (krancour) [inline docs/roadmap/planning.rst]: This link doesn't look right.

Comment (sgoings) [inline docs/roadmap/planning.rst]: Adjusted link at bottom of page.

**Comment by carmstrong** [comment]
:ship: Looks great!

**Pull request merged**